manifests/openshift/dev/kubevirt-scc.yaml.in

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: SecurityContextConstraints
+metadata:
+  name: kubevirt-scc
+  labels:
+    kubevirt.io: scc
+allowHostDirVolumePlugin: true
+allowHostIPC: true
+allowHostNetwork: true
+allowHostPID: true
+allowHostPorts: true
+allowPrivilegedContainer: true
+allowedCapabilities:
+  - '*'
+fsGroup:
+  type: RunAsAny
+readOnlyRootFilesystem: false
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+seccompProfiles:
+  - '*'
+supplementalGroups:
+  type: RunAsAny
+volumes:
+  - '*'
+users:
+  - system:serviceaccount:kube-system:kubevirt-privileged
+  - system:serviceaccount:kube-system:kubevirt-controller
+  - system:serviceaccount:kube-system:kubevirt-infra
```

Comment:
kubevirt-infra doesn't exist in latest KubeVirt

Reply:
I know. This change is compatible with older releases.

Reply:
why should you work on older releases? Especially since you consume the manifests from latest master?

Reply:
Was this added for 3.9 or did that already exist for 3.7?

Reply:
1. We need it for 3.7 and 3.9.
2. I'll remove `kubevirt-infra`.